packages/ui-components/src/components/action-button/action-button.tsx

```diff
@@ -52,7 +46,7 @@ export class KvActionButton implements IButton, IButtonEvents, IAnchor {
 	render() {
 		return (
 			<Host aria-disabled={this.disabled} onClick={this.onClickButton}>
-				<a
+				<div
```

Comment:
should we study in the future transform this into a button?

Reply:
Yes definitely! Let's do that on when we merge the action button components into a single one 👍